.github/workflows/ci.yaml

```diff
@@ -315,7 +300,7 @@ jobs:
       (needs.changes.outputs.go == 'true') || (needs.changes.outputs.baseimage == 'true') || ${{ github.event.workflow_dispatch.run_e2e == 'true' }}
     strategy:
       matrix:
-        k8s: [v1.26.15, v1.27.13, v1.28.9, v1.29.4, v1.30.0]
+        k8s: [v1.28.13, v1.29.8, v1.30.4, v1.31.0]
```

Comment:
Should we do this in a separate pr?

Reply:
I agree with James. This also affects our support matrix in the README.md.